ccloud/template_delta_configs/control-center-ccloud.delta

```diff
@@ -5,6 +5,7 @@ confluent.controlcenter.streams.sasl.mechanism=PLAIN
 confluent.controlcenter.streams.security.protocol=SASL_SSL
 bootstrap.servers=<CCLOUD_BOOTSTRAP_SERVER>
 confluent.controlcenter.streams.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<CCLOUD_API_KEY>" password\="<CCLOUD_API_SECRET>";
+confluent.metrics.topic.max.message.bytes=8388608
```

Comment:
@nicolasguyomar this file is auto-generated, so if this change needs to be made everywhere, then please make the corresponding change in the "source" at https://github.com/confluentinc/examples/blob/5.3.1-post/ccloud/ccloud-generate-cp-configs.sh